### ISSUE TITLE
git requirements prevent PyPI publishing

### ISSUE THREAD

**Issue by CBroz1** [datajoint/element-array-ephys#108]
In order to publish the latest release on PyPI, we would need to remove the following requirements

```
pyopenephys @ git+https://github.com/datajoint-company/pyopenephys.git
element-interface @ git+https://github.com/datajoint/element-interface.git
ibllib @ git+https://github.com/int-brain-lab/ibllib.git
```

1. element-interface is published and can be called from PyPI
2. our pyopenephys branch supports the latest version of open ephys naming convention, which the official release does not
3. find alternative package or remove support for the autocorrelegram implemented by @JaerongA via `ibllib`

**Comment by CBroz1** [comment]
Previously resolved by work from @iamamutt

**Comment by kabilar** [comment]
@CBroz1 I may be misunderstanding something, but the following is still in the `requirements.txt`.

  ```
  pyopenephys @ git+https://github.com/datajoint-company/pyopenephys.git
  ```

GitHub releases haven't been published to PyPI since `0.1.4`, so perhaps this is the reason?

**Comment by CBroz1** [comment]
Yes. While previous work removed the ibllib dependency, pyopenephys remains blocker. We have a meeting with maintainers planned to address this issue. Not sure why I closed this

**Comment by CBroz1** [comment]
Resolved by #125
